spec/features/eu_exit_business_readiness_request_spec.rb

```diff
@@ -78,7 +78,7 @@ def user_requests_update(details)
     within '#support_requests_eu_exit_business_readiness_request_public_sector_procurement_input' do
       check 'Civil government contracts'
     end
-    fill_in 'URL of content', with: '/some/base/path'
+    fill_in 'Which page is this about?', with: '/some/base/path'
```

Comment:
Should this contain the full label of `Which page is this about? Add the full URL`?

Reply:
I was being lazy. The tests match on the closest matching label, which is why sometimes you get a clash if the label appears in the form more than once.

I'm not a huge fan of matching on text. I'll see if I can update the text to match on the input id.

Reply:
@brucebolt Updated and fixed up.